### DISCUSSION TITLE
knowledge base usage

### DISCUSSION THREAD

**Discussion by yhl-186** [langgenius/dify#7604] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

If the markdown file has the URL of the image and I put it in the knowledge base, how do I get the app to display my image instead of answering the URL

**Comment by xxzxb** [comment]
me ,too.

**Comment by luogenhua** [comment]
mee too
